registry/securing-exposing-registry.adoc

```diff
@@ -6,12 +6,11 @@ toc::[]
 
 ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
 
-By default, the {product-title} registry is secured during cluster
-installation so that it serves traffic via TLS. A passthrough route is also
-created by default to expose the service externally.
+By default, the {product-title} registry is secured during cluster installation
+so that it serves traffic through TLS. Unlike previous versions of
+{product-title}, the registry is not exposed outside of the cluster at the time
+of installation.
```

Comment:
Do we know if this is true for OSD (since the ifdef includes `openshift-dedicated`)?

Reply:
No, I'm not sure. But I'm going to go ahead and remove the ifdefs for now since they're from legacy content.

Thanks!